controllers/grid/eventLog/EventLogGridCellProvider.inc.php

```diff
@@ -35,9 +48,47 @@ function getTemplateVarsFromRowColumn($row, $column) {
 			case 'date':
 				return array('label' => is_a($element, 'EventLogEntry') ? $element->getDateLogged() : $element->getDateSent());
 			case 'event':
-				return array('label' => is_a($element, 'EventLogEntry') ? $element->getTranslatedMessage() : $element->getPrefixedSubject());
+				return array('label' => is_a($element, 'EventLogEntry') ? $element->getTranslatedMessage(null, $this->_isCurrentUserAssignedAuthor) : $element->getPrefixedSubject());
 			case 'user':
-				return array('label' => is_a($element, 'EventLogEntry') ? $element->getUserFullName() : $element->getSenderFullName());
+				if (is_a($element, 'EventLogEntry')) {
+					$userName = $element->getUserFullName();
+					$reviewAssignmentDao = DAORegistry::getDAO('ReviewAssignmentDAO');
+
+					// Maybe anonymize reviewer log entries
+					$reviewerLogTypes = array(
+						SUBMISSION_LOG_REVIEW_ACCEPT,
+						SUBMISSION_LOG_REVIEW_DECLINE,
+						SUBMISSION_LOG_REVIEW_UNCONSIDERED,
+						SUBMISSION_LOG_REVIEW_FILE,
+						SUBMISSION_LOG_REVIEW_CANCEL,
+						SUBMISSION_LOG_REVIEW_REVISION,
+						SUBMISSION_LOG_REVIEW_RECOMMENDATION,
+					);
+					$params = $element->getParams();
+					if (in_array($element->getEventType(), $reviewerLogTypes) && isset($params['reviewAssignmentId'])) {
```

Comment:
What about existing log entries that don't have `reviewAssignmentId` available to them?

Reply:
Good catch! I missed this one because when "logged in as", the user gets set to the admin name.

Reply:
Oh! Also had to fix it so that it only applied to authors. :)